function_calling/streamlit/app.py

```diff
@@ -0,0 +1,129 @@
+import logging
```

Comment:
In another PR we need to do linting and type checking of this file too.

Reply:
yes, in my pending  list